src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py

```diff
@@ -328,3 +331,100 @@ def test_azure_vwan_p2s_gateway_routing_configuration(self):
+                     self.check('routingConfiguration.propagatedRouteTables.labels[0]', 'label1'),
+                     self.check('routingConfiguration.propagatedRouteTables.labels[1]', 'label2'),
+                     self.check('routingConfiguration.vnetRoutes.staticRoutes[0].name', self.kwargs['route']),
+                     self.check('routingConfiguration.vnetRoutes.staticRoutes[0].nextHopIpAddress', self.kwargs['next_hop'])
+                 ])
+
+        self.cmd('network vhub connection show '
+                 '-g {rg} '
+                 '-n {connection} '
+                 '--vhub-name {vhub}',
+                 checks=[
+                     self.check('provisioningState', 'Succeeded'),
+                     self.check('name', self.kwargs['connection']),
+                     self.check('routingConfiguration.associatedRouteTable.id', self.kwargs['route_table1']),
+                     self.check('length(routingConfiguration.propagatedRouteTables.ids)', 2),
+                     self.check('routingConfiguration.propagatedRouteTables.ids[0].id', self.kwargs['route_table1']),
+                     self.check('routingConfiguration.propagatedRouteTables.ids[1].id', self.kwargs['route_table2']),
+                     self.check('length(routingConfiguration.propagatedRouteTables.labels)', 2),
+                     self.check('routingConfiguration.propagatedRouteTables.labels[0]', 'label1'),
+                     self.check('routingConfiguration.propagatedRouteTables.labels[1]', 'label2'),
+                     self.check('routingConfiguration.vnetRoutes.staticRoutes[0].name', self.kwargs['route']),
+                     self.check('routingConfiguration.vnetRoutes.staticRoutes[0].nextHopIpAddress', self.kwargs['next_hop'])
+                 ])
+
+        self.cmd('network vhub connection list '
+                 '-g {rg} '
+                 '--vhub-name {vhub}',
+                 checks=[
+                     self.check('length(@)', 1)
+                 ])
+
+        self.cmd('network vhub connection delete '
+                 '-g {rg} '
+                 '-n {connection} '
+                 '--vhub-name {vhub} '
+                 '-y',
+                 checks=[])
+
+        with self.assertRaisesRegexp(SystemExit, '3'):
+            self.cmd('network vhub connection show '
+                     '-g {rg} '
+                     '-n {connection} '
+                     '--vhub-name {vhub}')
+
+    @record_only()
+    @ResourceGroupPreparer(name_prefix='cli_test_azure_vwan_vhub_get_effective_routes')
+    def test_azure_vwan_vhub_get_effective_routes(self):
+        self.kwargs.update({
+            'rg': 'azure-cli-test-rg',
+            'vhub': 'yu-vhub1',
+            'resource_type': 'P2SConnection',
+            'resource_id': '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/azure-cli-test-rg/providers/Microsoft.Network/p2sVpnGateways/p2svpngateway/p2sConnectionConfigurations/myconnectionconfig'
```

Comment:
Is it better to add some description about this resource so that we can re-build it later?

Reply:
will add